Src/PChecker/CheckerCore/SystematicTesting/Traces/ScheduleTrace.cs

```diff
@@ -51,7 +51,7 @@ internal ScheduleTrace(string[] traceDump)
 
             foreach (var step in traceDump)
             {
-                if (step.StartsWith("--") || step.Length == 0)
+                if (step.StartsWith("--") || step.Length == 0 || step.StartsWith("//"))
```

Comment:
Can you explain this change?

Reply:
To ignore any comment lines starting with "//" from the .schedule file.
.schedule printed with PSym contains some comment lines.
Let's have a common format for all engines.